core/team-members.rst

```diff
@@ -45,7 +45,12 @@ I am Hemang, I'm a Computer Science and Engineering undergrad, I like
 programming in Python, JavaScript and I also love making Android & Web projects.
 I like to learn new stuff everyday and to share my knowledge whenever I can.
 
-`Maxim Alexandru https://github.com/Nosferatul`_
+`Maxim Alexandru <https://github.com/Nosferatul>`_
```

Comment:
heh unrelated change :P

Reply:
His name was showing the full link to his profile unlike the rest and I'm a big fan of symmetry so couldn't resist :wink:

Reply:
is it okay if we keep or do you want me to amend the commit?

Reply:
amend and new commit with this change

Reply:
done @Makman2  :+1: